scripts/tests_coverage.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Butterfly build root
+BUTTERFLY_BUILD_ROOT=$1
+
+client_obj_dir=$BUTTERFLY_BUILD_ROOT/api/client/CMakeFiles/api_client.dir/
+server_obj_dir=$BUTTERFLY_BUILD_ROOT/api/server/CMakeFiles/api_server.dir/
+
+# Test Butterfly build root
+if [ ! -d $client_obj_dir -o ! -d $server_obj_dir ]; then
+    echo "Please set butterfly's build root as parameter"
```

Comment:
echo "either '$client_obj_dir' or '$server_obj_dir' is not a directory have you set BUTTERFLY_BUILD_ROOT ?"
would be more explicit